internal/controllers/rest_controller.go

```diff
@@ -71,24 +131,77 @@ func (rc RESTController) OIDCProviderCallback(w http.ResponseWriter, r *http.Req
 	safe, err := rc.oidcService.IsFlowSecure(stateProvider, token)
 	if !safe {
 		rc.logger.Error(err)
```

Comment:
same as above

Reply:
we also already have a custom error coming from IsFlowSecure